code/modules/mob/living/living_defense.dm

```diff
@@ -501,7 +501,8 @@
 		taste(source)
 
 	var/touch_protection = (methods & VAPOR) ? get_permeability_protection() : 0
-	SEND_SIGNAL(source, COMSIG_REAGENTS_EXPOSE_MOB, src, reagents, methods, volume_modifier, show_message, touch_protection)
+	if (source)
```

Comment:
why?

Reply:
Bees and venom don't have a source holder. Trying to send a signal to null runtimes.